foruma-frontend/src/stories/Header.stories.js

```diff
@@ -0,0 +1,28 @@
+import MyHeader from './Header.vue';
+
+export default {
+  title: 'Example/Header',
```

Comment:
Codacy has a fix for the issue: [Replace `'Example/Header'` with `"Example/Header"`](https://app.codacy.com/gh/agabani/foruma/pullRequest?prid=7388752)
```suggestion
  title: "Example/Header",
```